components/docs-chef-io/content/automate/ha_onprim_deployment_with_customer_managed_deployment.md

```diff
@@ -48,15 +48,33 @@ This section will discuss deploying Chef Automate HA on-premises machines with a
 
 ## Verify
 
-1. Verify the configuration file.
+### Prerequisites
 
-    ```bash
-    sudo chef-automate verify -c config.toml
-    ```
+#### * Directory Structure
+
+- A directory must exist on all cluster nodes at `/home/<ssh_user name>/`.
+- The should be specified in the 'config.toml' configuration file under the `ssh_user` key.
+
+#### * Permission Requirements
+
+- The specified SSH user must have:
+  - Read (r), write (w), and execute (x) permissions.
+  - Ownership of the directory.
+
+#### * Example
+
+If `config.toml` contains `ssh_user = "user1"`, then every node must have the `/home/user1/` directory structure with permissions `drwx------ user1 user1 /home/user1`.
+
+
+Verify the configuration file.
+
+```bash
+sudo chef-automate verify -c config.toml
+```
 
-    To know more about config verify, you can check [Config Verify Doc page](/automate/ha_verification_check/).
+To know more about config verify, you can check [Config Verify Doc page](/automate/ha_verification_check/).
```

Comment:
Replace it with: To learn more about Config Verify, check the [Config Verify Documentation](https://github.com/automate/ha_verification_check/).